### PR TITLE
Change license to Apache 2.0, update repo to follow new standards

### PR DESCRIPTION
**Pull request by ramonguiu**
### What are you changing/fixing?

Updated license
Added code of conduct and contributing docs
Updated README to match the new templates and moved troubleshooting section to its own document

### Does your Pull Request introduce breaking changes?

No

### Do the users need to upgrade immediately to the new version?

No

### Do you introduce new dependencies on other libraries?

No

## Checklist: before you submit

- [ ] apply the labels that best suit the context of your Pull Request.
- [ ] include unit or integration testing for your changes/additions.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Contributing
+
+Contributions are always welcome. Before contributing please read the
+[code of conduct](./CODE_OF_CONDUCT.md) and [search the issue tracker](issues); your issue may have already been discussed or fixed in `main`. To contribute,
```
Comment (davidbrota) [inline CONTRIBUTING.md:4]: the (issues) link gives a 404. Shouldn't it be (../../issues)?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
-# NR-JMX
-New Relic's JMX fetcher, a simple tool for extracting data out of any application exposing a JMX interface.
+[![Community Project header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Community_Project.png)](https://opensource.newrelic.com/oss-category/#community-project)
+
+# New Relic JMX fetcher
+
+A simple tool for extracting data out of any application exposing a JMX interface.
```
Comment (theletterf) [inline README.md:5]:
```suggestion
A tool for extracting data out of any application exposing a JMX interface.
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (theletterf) [inline CONTRIBUTING.md:4]:
```suggestion
[code of conduct](./CODE_OF_CONDUCT.md) and [search the issue tracker](../../issues); your issue may have already been discussed or fixed in `main`. To contribute,
```

File: README.md [hunk 3]
```diff
@@ -13,58 +16,15 @@ Ie: `yum install nrjmx`
...
+### New Relic JMX integration (nri-jmx)
 
-`nrjmx` is *not* bundled within the `nri-jmx` package. But, it's declared as a dependency. 
+`nrjmx` is *not* bundled within the `nri-jmx` package. But, it's declared as a dependency.
 
 So while installing `nri-jmx` if you have `nrjmx` already installed it keeps the installed version, otherwise it'll try to get the latest `nrjmx` release.
 
+## Getting Started
 
-## Custom Build
```
Comment (theletterf) [inline README.md:23]: @ramonguiu Can we move this information to a new .md file in a /docs/ subfolder?

File: README.md [hunk 4]
```diff
@@ -121,103 +85,41 @@ To extract data from this application:
 ```bash
 $ ./bin/nrjmx -hostname localhost -port 1689 -uriPath "org.opends.server.protocols.jmx.client-unknown" -username user -password pwd
 ```
+### Troubleshooting
 
----
```
Comment (theletterf) [inline README.md:125]: @ramonguiu Same here, before we decide whether to move this to the docs site or not.

Comment (theletterf) [reply README.md:125]: Ah, you moved this to TROUBLESHOOT.MD?

File: TROUBLESHOOT.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
+# Troubleshooting via jmxterm
```
Comment (carlosroman) [inline TROUBLESHOOT.md:1]: Nice 👏

**Pull request merged**